docs/build.gradle

```diff
@@ -0,0 +1,49 @@
+buildscript {
+    ext.kotlin_version = '1.2.31'
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+
+group 'io.improbable'
+version '1.0-SNAPSHOT'
+
+apply plugin: 'java'
+apply plugin: 'kotlin'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+dependencies {
+    compile project(':keanu-project')
+
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+}
+
+compileKotlin {
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+}
+compileTestKotlin {
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+}
+sourceSets {
```

Comment:
This sourceSets block can be removed if we move the java source to the standard `src/main/java` instead.

Reply:
Yeah I guess the reason I used this is so that we don't have `src/markdown` and `src/main` within the same directory. I personally think it makes more semantic sense to have a directory with languages as all the subdirectories rather than just some of them. Am willing to change it back if you think this is more confusing though @gordoncaleb?